app-misc/lirc/lirc-0.9.4a-r2.ebuild

```diff
@@ -22,7 +22,7 @@ fi
 LICENSE="GPL-2"
 SLOT="0"
 KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
-IUSE="doc static-libs X audio irman ftdi inputlirc iguanair systemd usb"
```

Comment:
My apologies for the confusion here - I do not want to regress and remove `static-libs` if it's not problemic, so I've added it back in. Thank you once again for you help!